.github/workflows/ci.yml

```diff
@@ -218,6 +218,9 @@ jobs:
         env:
           RUST_BACKTRACE: 1
           AUTH0_FLOX_DEV_CLIENT_SECRET: "${{ secrets.MANAGED_AUTH0_FLOX_DEV_CLIENT_SECRET }}"
+          FLOX_CI_RUNNER: "github-${{ matrix.os }}"
+        # Skip containerize tests on the macOS runner because we can't double nest virtualization
+        if: ${{ !((matrix.os == 'macos-14-xlarge') && (matrix.test-tags == 'containerize'))}}
```

Comment:
blocking: I think this conditional needs to be on the whole job, because currently both unit tests and integration tests can get skipped, which means we're running a pointless job https://github.com/flox/flox/actions/runs/12680951293/job/35343701195?pr=2514

Reply:
TIL that `matrix.exclude` exists